src/app/quicklook/page.tsx

```diff
@@ -88,7 +86,7 @@ export default function QuickLook() {
           if (!res.ok) throw new Error(await res.text());
           const returndata = await res.json();
           json = returndata.data;
-          setData(json as unknown as JSONData);
+          setData(json as unknown as HealthJSONData);
```

Comment:
I'm seeing a lot of bypassing of Typescript typing. Is there a particular reason why? I'm assuming you already tried partial and full typing since HealthJSONData is imported from types. Feel free to reply to these comments btw.

Reply:
Typescript suggested that returning JSON does not necessarily contain the fields of HealthJSONData. This is why I created jsonValidator to verify the JSON to ensure the bypass is safe.

Reply:
Yea my main concern was as long as it's safe it should be fine. Although that is strange Typescript suggested that. I guess since we aren't using type-safe ORM there's a bit of extra steps and ambiguity